### ISSUE TITLE
<PopoverMenu.Divider /> changes background on hover

### ISSUE THREAD

**Issue by marik22312** [wix/wix-style-react#5781]
# 🐛 Bug Report

### 🏗 Relevant Components

`<PopoverMenu.Divider />`

### 😯 Current Behavior

When hovering the divider, it's background color is changed
### 🤔 Expected Behavior

Dividers should be interactable and shouldn't change background color 

### 👣 Steps to Reproduce

https://www.wix.com/pages/wix-style-react/?path=/story/introduction-playground--playground&snippet=7db731b6-2324-412d-a2b7-074e81670474

  - **WSR version**: `8.35`

### 👀 Severity

- Low

**Comment by EgoziE** [comment]
There's also an accessibility issue here, can't use arrows to go over items.
